### ISSUE TITLE
tested memory chip, bugfix

### ISSUE THREAD

**Issue by cpdef** [jogag/digiline-stuff#7]
I have tested the Memory chip, removed some bugs and then retested.
Then i didn't found errors anymore.
There were 2 errors:
1.`digiline:receptor_send(pos, digilines.rules.default, channel, OK_MSG)`
to `digiline:receptor_send(pos, digiline.rules.default, channel, OK_MSG)`
(line 48, changed second digilines... to digiline...)
2.same error at line 62:
`digiline:receptor_send(pos, digilines.rules.default, channel, OK_MSG)`
to
`digiline:receptor_send(pos, digilines.rules.default, channel, OK_MSG)`
3.see last issue, changed at line 54 "[2]" to "[1]"
should i send pull request?

**Comment by p5yg3n** [comment]
Definitely open a PR. Even if it doesn't get merged people have a better view of the fix/changes if they want to apply it themselves.

**Comment by cpdef** [comment]
Can't send a Pull request, because i can't create a new branch.
I send the code here ok?

**Comment by p5yg3n** [comment]
Yeah that's fine, the first post is good enough then :+1:. I'm going to reference the issue you speak of in point 3: #6

**Comment by cpdef** [comment]
and the other bugs? with "digilines" instead of "digiline"? They let the server crash, if u use the "ERASE" command

**Comment by p5yg3n** [comment]
Wow I have never actually used any chips. I can not believe how broken they are :laughing:. Thanks for spotting these bugs.

Hopefully this mod is not dead and these fixes will be merged.

Oh and in order to make a Pull Request you have to:
1. Fork the Repository
2. Make changes to **YOUR** Repository (Does not matter what Branch)
3. Compare the changes / Make a Pull Request (Has to be the Branch you made changes to)

It took me a couple tries to get it right. If something goes wrong then delete the Repository and start over. There are plenty of tutorials as well, if I was not helpful.

**Comment by cpdef** [comment]
ok, if they won't be merget i open a new digiline mod, because I'am programming at this time a 
scan adn copy device for digilines
